phoenix-core/src/it/java/org/apache/phoenix/calcite/CalciteIT.java

```diff
@@ -1343,6 +1343,28 @@ public void initTable() throws Exception {
                         {7L, "0000000005", "T5", "0000000005", "S5"},
                         {8L, "0000000006", "T6", "0000000006", "S6"}})
                 .close();
+
```

Comment:
I remember you said you ran into a problem using PhoenixServerProject, but it's PhoenixClientProject here again. Does it oscillate between the two? Could you please try {{start(false, 1f)}} instead?